mike/themes/material/js/version-select.js

```diff
@@ -1,49 +1,46 @@
 window.addEventListener("DOMContentLoaded", function() {
-  // This is a bit hacky. Figure out the base URL from a known CSS file the
-  // template refers to...
-  var ex = new RegExp("/?assets/fonts/material-icons.css$");
-  var sheet = document.querySelector('link[href$="material-icons.css"]');
-
-  var ABS_BASE_URL = sheet.href.replace(ex, "");
-  var CURRENT_VERSION = ABS_BASE_URL.split("/").pop();
-
+  var VERSION = window.location.pathname.split('/')[1];
```

Comment:
We can't rely on the version component being the first subdirectory in the URL after the initial `/`. This would fail for most common uses on Github: `https://user.github.io/project/1.0/...`

Reply:
Indeed, I'm currently using a custom domain and the docs live directly under the root of the hostname. But it won't work for other scenarios and indeed it is not as trivial to detect under which situation we're in. :)